Assets/MixedRealityToolkit.Examples/Demos/UX/Collections/Scripts/GridObjectLayoutControl.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+using UnityEngine.UI;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Provides functions to control layout of GridObjectCollection as well
+    /// as to output positions of child controls to help with building
+    /// GridObjectCollectionTests.
+    /// </summary>
+    public class GridObjectLayoutControl : MonoBehaviour
+    {
+        [Tooltip("Point this at the GridObjectCollection to control.")]
+        public GridObjectCollection grid;
+        [Tooltip("Optional text field to output the layout of control.")]
+        public TMPro.TextMeshPro text;
+
+        /// <summary>
+        /// Change the grid collection's layout to the next one in order.
+        /// </summary>
+        public void NextLayout()
+        {
+            GridObjectCollection.AnchorType cL = grid.Anchor;
```

Comment:
>cL [](start = 44, length = 2)

What are folk's thoughts on the usage of abbreviations? I've seen that shorthand like this lead to confusion because it creates additional cognitive overload of having to think about "what does cL" stand for? This overhead isn't that useful  (i.e. they shouldn't have to mentally expand out the abbreviation, especially when something like 'currentAnchorType' more clearly shows intent/what it is.

Here's an example (albiet C++) guideline where abbreviations are warned against:

[https://google.github.io/styleguide/cppguide.html#General_Naming_Rules](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules)

If folks agree I think I'd like to just add this to our guide as well. I believe that folks on the OS side also favored against hungarian for similar reasons.

Reply:
Oh great point, thanks! I forgot to remove that